src/server.js

```diff
@@ -0,0 +1,17 @@
+const http = require('http');
+const url =require('url');
+const {airportInformation} = require('./airportDetailsProvider.js');
+const host = 'localhost';
+const port = 3000;
+const server = http.createServer((request, response) => {
+    response.writeHead(200, {'Content-Type': 'text/html'});
+    const queryParameter = url.parse(request.url, true).query;
+    let code = queryParameter['airport-code'];
+    let airportObj = airportInformation(code);
+    //const aiprortCodeReq = queryParameter.airport-code ;
+    response.end(JSON.stringify(airportObj));
```

Comment:
No error handling, the function `airportInformation(code)` throws error. Here it is not caught and handled, any endpoint other than http://localhost:3000/airport-details?airport-code=DEL including a wrong code case like http://localhost:3000/airport-details?airport-code=WWW will cause server to crash 💣💥